<?xml version="1.0"?>
<!-- 
    Bringup launch file for Scout sensor
 -->
<launch>
    <node pkg="rslidar_sdk" name="rslidar_sdk_node" type="rslidar_sdk_node" output="screen">
        <param name="config_path" value=""/>
        <remap from="/rslidar_points" to="/velodyne_points"/>
    </node>

    <node pkg="ros_indigosdk" type="ros_indigosdk_node" name="ros_indigosdk_occam"/>

</launch>